### PR TITLE
[YUNIKORN-1553] updateAllocatedAndPendingResourceMetrics should support all queue types

### PR DESCRIPTION
**Pull request by zhuqi-lucas**
### What is this PR for?
   
updateAllocatedAndPendingResourceMetrics should support all queue types

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/YUNIKORN-1553
* Put link here, and add [YUNIKORN-*Jira number*] in PR title, eg. `[YUNIKORN-2] Gang scheduling interface parameters`

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: pkg/scheduler/objects/queue.go [hunk 1]
```diff
@@ -1482,16 +1482,14 @@ func (sq *Queue) updateMaxResourceMetrics() {
 
 // updateAllocatedAndPendingResourceMetrics updates allocated and pending resource metrics if this is a leaf queue.
 func (sq *Queue) updateAllocatedAndPendingResourceMetrics() {
```
Comment (wilfred-s) [inline pkg/scheduler/objects/queue.go:1484]: please update comment also

**Review by wilfred-s — CHANGES_REQUESTED** [review]

File: pkg/scheduler/objects/queue.go (hunk 1 above)
Comment (zhuqi-lucas) [reply pkg/scheduler/objects/queue.go:1484]: Sure @wilfred-s , sorry for missing this comment change.

**Comment by pbacsko** [comment]
+1

**Review by pbacsko — APPROVED** [review]
+1

**Pull request closed without merging**